src/components/shared/Slider/Slider.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const Slider = ({ min, max, step, defaultValue, handleChange }) => {
+  // TODO: add styles
+  return (
+    <input
+      style={{ marginTop: '100px', padding: '0' }}
+      type="range"
```

Comment:
Does it work with all browsers that we want to support?

Reply:
I think that yes, At least I have tested it in Firefox, Chrome and Edge and it works fine. In Sites they are using the same element.